spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/write/context/BigQueryDataSourceWriterModule.java

```diff
@@ -76,9 +76,10 @@ public BigQueryDirectDataSourceWriterContext provideDirectDataSourceWriterContex
         tableConfig.getKmsKeyName(), // needs to be serializable
         tableConfig.isWriteAtLeastOnce()
             && !mode.equals(
-                SaveMode.Overwrite) // writeAtLeastOnce mode is currently not supported in OverWrite
+                SaveMode
```

Comment:
Redundant change

Reply:
This was changed because of formatting I think